ios/Notification Service/NotificationService.swift

```diff
@@ -1,13 +1,18 @@
 import UserNotifications
 import CioMessagingPush
+import CioTracking
+
 class NotificationService: UNNotificationServiceExtension {
    
       override func didReceive(_ request: UNNotificationRequest,
                                withContentHandler contentHandler:
                                @escaping (UNNotificationContent) -> Void) {
         
+        CustomerIO.initialize(siteId: "94541e1bbff594682089", apiKey: "914ea21ebfa87bf771f4", region: .US) { config in
```

Comment:
Shouldn't we keep keys in ignored files and use from there? 👀

Reply:
Oops! My bad 🤣 
Thanks for noticing this !

Reply:
This doesn't look like a quick change. I will have to update CI and secret keys. Will have to open another PR to fix this. I have removed the keys and adding a note in Todo marker.